contracts/btc-finality/src/multitest/suite.rs

```diff
@@ -145,6 +151,13 @@ pub struct Suite {
 }
 
 impl Suite {
+    pub fn to_consumer_addr(&self, bbn_addr: &Addr) -> Result<Addr, StakingApiError> {
+        let babylon_prefix = bbn_addr.as_str().split('1').collect::<Vec<_>>()[0];
+        let consumer_prefix = self.babylon.as_str().split('1').collect::<Vec<_>>()[0];
```

Comment:
Curious why `split('1')`?

Reply:
Because all bech32 addresses have a '1' after the prefix?

This is just a test helper method, didn't check the exact bech32 spec for this.